Makefile

```diff
@@ -87,6 +88,20 @@ searx.brand:
 	$(Q)echo "DOCS_URL = '$(DOCS_URL)'" >> searx/brand.py
 	$(Q)echo "PUBLIC_INSTANCES = 'https://searx.space'" >> searx/brand.py
 
+# build themes
+# ------------
+
+PHONY += themes themes.oscar themes.simple
+themes: themes.oscar themes.simple
+
+themes.oscar:
+	$(Q)echo '[!] Grunt build : oscar theme'
+	$(Q)grunt --gruntfile  "searx/static/themes/oscar/gruntfile.js"
+
+themes.simple:
+	$(Q)echo '[!] Grunt build : simple theme'
+	$(Q)grunt --gruntfile  "searx/static/themes/simple/gruntfile.js"
```

Comment:
```sh
export PATH="$(npm bin)":$PATH
``` 
is missing. See https://github.com/asciimoo/searx/blob/master/manage.sh#L118-L123

Reply:
Yes, I dropped it since it was not needed (at least in my tests).  Do you know for what this PATH extension is needed?

Reply:
grunt and some other npm packages are required.

```./manage.sh npm_packages``` installs these requirements locally, also it installs required dependencies for the simple and oscar theme:
https://github.com/asciimoo/searx/blob/da56bda26542de5697691ff80dac6df876fb366b/manage.sh#L122-L136

Then the [npm_path_setup](https://github.com/asciimoo/searx/blob/da56bda26542de5697691ff80dac6df876fb366b/manage.sh#L117-L120) function setups the PATH to use the local grunt install.

Not all user will have grunt. Actually using the grunt from the Linux distribution can be a problem since a different grunt version can behave differently with the same gruntfile.js